qiita_db/test/test_metadata_template.py

```diff
@@ -652,6 +775,133 @@ def test_create(self):
                ['2.Sample3', "Value for sample 3"]]
         self.assertEqual(obs, exp)
 
+    def test_create_int_prefix(self):
+        """Creates a new SampleTemplate"""
+        st = SampleTemplate.create(self.metadata_int_pref, self.new_study)
+        # The returned object has the correct id
+        self.assertEqual(st.id, 2)
+
+        # The relevant rows to required_sample_info have been added.
+        obs = self.conn_handler.execute_fetchall(
+            "SELECT * FROM qiita.required_sample_info WHERE study_id=2")
+        # sample_id study_id physical_location has_physical_specimen
+        # has_extracted_data sample_type required_sample_info_status_id
+        # collection_timestamp host_subject_id description
+        exp = [["2.12.Sample1", 2, "location1", True, True, "type1", 1,
```

Comment:
@wasade Note that storing the study id and the sample id in two different columns will bring us back to the original design, where we actually found a lot of issues and we end up doing a lot changes in the DB to just aggregate the study id to the sample id (that was done on september, and I suffered it so I remember :tired_face: )